src/coreComponents/linearAlgebra/interfaces/hypre/HypreSolver.cpp

```diff
@@ -311,6 +311,12 @@ void HypreSolver::solve_krylov( HypreMatrix & mat,
 {
   Stopwatch watch;
 
+#if defined(USE_CUDA)
+  HYPRE_ExecutionPolicy default_exec_policy = HYPRE_EXEC_DEVICE;
+  hypre_HandleDefaultExecPolicy(hypre_handle()) = HYPRE_EXEC_DEVICE;
+  hypre_HandleSpgemmUseCusparse(hypre_handle()) = 1;
```

Comment:
@oseikuffuor1 @mquanbui 
...so I added them here too.

Reply:
@rrsettgast you don't need to specify it in both places, and the way you have it in the initialize function should be fine. Having it here is fine too, but not necessary. It only needs to be set prior to making calls to hypre that would be executed on the GPU.